libevm/VMFactory.cpp

```diff
@@ -105,6 +103,8 @@ void setVMKind(const std::string& _name)
                 "loading " + _name + " failed"));
     }
     g_kind = VMKind::DLL;
+
+    cnote << "Loaded EVMC module: " << g_evmcDll->name() << " " << g_evmcDll->version() << " (" << _name << ")";
```

Comment:
I guess this will not be formatted properly not only in testeth but in aleth either, because `setupLogging` is called after handling all command line parameters

Reply:
I don't see any easy fix for it other than just output to `cout`